ngraph/test/type_prop/experimental_detectron_roi_feature_extractor.cpp

```diff
@@ -169,3 +169,153 @@ TEST(type_prop, detectron_roi_feature_extractor_intervals)
         ASSERT_TRUE(roi->get_output_partial_shape(0).same_scheme(ref_out_shape));
     }
 }
+
+TEST(type_prop, detectron_roi_feature_extractor_intervals_1)
+{
+    Attrs attrs;
+    attrs.aligned = false;
+    attrs.output_size = 14;
+    attrs.sampling_ratio = 2;
+    attrs.pyramid_scales = {4, 8, 16, 32};
+
+    struct Shapes
+    {
+        PartialShape input_shape;
+        Dimension channels[4];
+        Dimension first_dims[4];
+    };
+
+    const auto dyn_dim = Dimension::dynamic();
+
+    std::vector<Shapes> shapes = {
```

Comment:
I think will be better to use the standard approach and create parametrized test.

Reply:
Please, give me link to an example of such `type_prop.*` tests.

Reply:
You can try to use `TEST_P` as a keyword to search in the OpenVINO sources, I think you will get a lot of results or you can use [google](https://www.google.com/search?q=gtest+parameterized+test+example) to find more examples.

Reply:
Done.